grunt.js

```diff
@@ -219,72 +218,59 @@ module.exports = function(grunt) {
       }
     },
     browserify: {
+      options: {
+        browserifyOptions: {
+          debug: true,
+          standalone: 'videojs'
+        },
+        banner: license,
+        plugin: [
+          ['browserify-derequire']
+        ],
+        transform: [
+          require('babelify').configure({
+            sourceMapRelative: './',
+            loose: ['all']
+          }),
+          ['browserify-versionify', {
+            placeholder: '__VERSION__',
+            version: pkg.version
+          }],
+          ['browserify-versionify', {
+            placeholder: '__VERSION_NO_PATCH__',
+            version: version.majorMinor
+          }]
+        ]
+      },
       build: {
         files: {
           'build/temp/video.js': ['src/js/video.js']
-        },
-        options: {
-          browserifyOptions: {
-            debug: true,
-            standalone: 'videojs'
-          },
-          banner: license,
-          plugin: [
-            [ 'browserify-derequire' ]
-          ],
-          transform: [
-            require('babelify').configure({
-              sourceMapRelative: './src/js',
-              loose: 'all'
-            }),
-            ['browserify-versionify', {
-              placeholder: '__VERSION__',
-              version: pkg.version
-            }],
-            ['browserify-versionify', {
-              placeholder: '__VERSION_NO_PATCH__',
-              version: version.majorMinor
-            }]
-          ]
         }
       },
-      test: {
-        files: {
-          'build/temp/tests.js': [
-            'test/globals-shim.js',
-            'test/unit/**/*.js'
-          ]
+      watch: {
+        options: {
+          watch: true,
+          keepAlive: true
         },
+        files: {
+          'build/temp/video.js': ['src/js/video.js']
```

Comment:
I've been using the side-by-side view of the diffs lately. Definitely seems to help for diffs like this.